### PR TITLE
BXMSDOC-7504 Added new EDITING PROCESS VARIABLE DATA TYPES section

### PR DESCRIPTION
**Pull request by michelehaglund**
Original jira: https://issues.redhat.com/browse/BXMSDOC-7504
Rendered output:
jBPM: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-7504_COMM/#proc-edit-process-variables_business-processes

PAM: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-7504/#proc-edit-process-variables_business-processes

### CODE REVIEW

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[id='proc-edit-process-variables_{context}']
+= Editing process variable data types
```
Comment (MarianMacik) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc:2]: Actually the feature is about correctly changing the value of the variables. Data types are not changed at all. So can we rephrase this? Thanks.

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+After starting a process instance, you can edit process variable data types in {CENTRAL}.
+
+.Prerequisites
+* You have created a project in {CENTRAL} and have started a process instance.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. Select the *Process Variables* tab and click *Edit* for the variable name that you want to edit.
+. Add or change the *Variable Value* and click *Save*.
```
Comment (MarianMacik) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc:12]: Here it correctly states, that we change the `Variable Value`, not data type.

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[id='proc-edit-process-variables_{context}']
+= Editing process variable data types
+
+After starting a process instance, you can edit process variable data types in {CENTRAL}.
```
Comment (MarianMacik) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc:4]: Same here.

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+.Prerequisites
+* You have created a project in {CENTRAL} and have started a process instance.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. Select the *Process Variables* tab and click *Edit* for the variable name that you want to edit.
+. Add or change the *Variable Value* and click *Save*.
+
+.Variable data type edit button in {CENTRAL}
```
Comment (MarianMacik) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-edit-process-variables.adoc:14]: Same here.

**Review by MarianMacik — APPROVED** [review]
Looks good now, thanks!

**Comment by krisv** [comment]
@michelehaglund Can we mention that this works for booleans, floats, integers and enums (but not for others like lists or custom objects)?

**Pull request merged**